.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 jobs:
   validate:
     docker:
-      - image: trussworks/circleci-docker-primary:e66fbea875bcb788b29b1b5f59142e8231961ec5
+      - image: trussworks/circleci:29ab89fdada1f85c5d8fb685a2c71660f0c5f60c
```

Comment:
why is this one not `6986bb9022e5a83599feb66a7128a2d0fa12732a`?

Reply:
This one is the latest version of the `circleci` image which includes the circleci cli.